madatv_android_settings.json

```diff
@@ -10,5 +10,17 @@
       "TYPE": "jobType.PASSTHROUGH",
       "SYNTAX": "su -c 'settings put global heads_up_notifications_enabled 0'"
     }
+  ],
+  "ATV - HDMI control: enable": [
+    {
+      "TYPE": "jobType.PASSTHROUGH",
+      "SYNTAX": "su -c 'settings put global hdmi_control_enabled 1'"
+    }
+  ],
+  "ATV - HDMI control: disable": [
+    {
+      "TYPE": "jobType.PASSTHROUGH",
+      "SYNTAX": "su -c 'settings put global hdmi_control_enabled 0'"
+    }
```

Comment:
i dont think this is needed. 1.2.2 disabled hdmi-cec and there is no use case for needing it in scanners

Reply:
If we assume that everybody updates the ROM frequently, then you're right. I'll remove this section.

Reply:
we wont be putting a job for everything the rom updates. either they update or they can figure it out :shrug:

Reply:
the alternative would be to have a job for each thing done in 16mad, which is actually a valid thought... one could even make a single job that could do everything the mad rom does if they wanted. if that is the direction you want to go, then i guess i see no reason why not...

Reply:
Honestly I didn't really think that through :) Let's leave it out for now..